treatment-service/services/metric_service.go

```diff
@@ -48,16 +57,22 @@ func NewMetricService(cfg config.Monitoring, localStorage *models.LocalStorage)
 	svc := &metricService{
 		Kind:         cfg.Kind,
 		LocalStorage: localStorage,
+		MetricLabels: cfg.MetricLabels,
```

Comment:
The `metricService` struct now keeps a copy of the metric labels found in the config so that the plugin (which does not store a copy of the monitoring configs, unlike the global treatment service) gets to use them for logging.